ci-operator/config/openshift-kni/performance-addon-operators/openshift-kni-performance-addon-operators-master.yaml

```diff
@@ -105,7 +105,7 @@ tests:
       OO_CHANNEL: "4.5"
       OO_INSTALL_NAMESPACE: openshift-performance-addon
       OO_PACKAGE: performance-addon-operator
-      OO_TARGET_NAMESPACES: '!install'
+      OO_TARGET_NAMESPACES: '!all'
```

Comment:
why also for 4.5?

Reply:
@cynepco3hahue , can you respond? I asked the same question.

Reply:
Is depending on how the PAO using 4.5 channel is installed. If is a mock and it uses the tested PAO image with the same CSV we obviously need it, otherwise, if a PAO 4.5 standard image is used, we don't. (but it can't hurt)

Reply:
well, let it be